src/containers/dashboard/home/index.js

```diff
@@ -49,10 +51,10 @@ const Home = props => [
   </List>,
   <h2 key="3">{props.user.name}</h2>,
   <List key="4">
-    <ListItem clickable={true} onClick={props.payment}>
+    {!props.user.paid ? <ListItem clickable={true} onClick={props.payment}>
       <h3>Payer ma place</h3>
       <span>Paye ta place et récupère ton billet d'entrée ! Obligatoire pour les tournois</span>
-    </ListItem>
+    </ListItem> : null}
```

Comment:
Maybe show instaed of Null, show a non clickable ListItem with `Ta place est déjà payée :)`